packages/InteractiveIcon/Limited.jsx

```diff
@@ -4,22 +4,12 @@ import styled, { ThemeProvider } from 'styled-components'
 
 import { colorShark, colorTelusPurple, colorWhite, colorAccessibleGreen } from '@tds/core-colours'
 
-export const StyledLimitedInteractiveIconSVG = styled.svg(
-  ({ theme }) => ({ fill: theme.iconColor }),
-  ({ animationDirection }) => ({
-    width: '24px',
-    height: '24px',
-    zIndex: '2',
-    transition: 'transform 150ms ease-in-out',
-    '@media (prefers-reduced-motion: reduce)': {
-      transition: 'none',
-    },
-    [`:hover, :focus, :active`]: {
-      transform: `translate${
-        animationDirection === 'up' || animationDirection === 'down' ? 'Y' : 'X'
-      }(${animationDirection === 'up' || animationDirection === 'left' ? '-' : ''}4px)`,
-    },
-  })
+import animations from './shared/animations'
+import StyledInteractiveIconSVG from './shared/StyledInteractiveIconSVG'
+
+export const StyledLimitedInteractiveIconSVG = styled(StyledInteractiveIconSVG)(
+  animations.reduceMotion,
+  animations.translate
```

Comment:
This took me a while to understand: a single Limited SVG icon would consume `StyledLimitedInteractiveIconSVG` that consumes `animations.translate` that accepts an `animationDirection` prop. That is three levels of abstraction. 

Do you think `animations.translate` should only live here on line 12 since it's only being used by the Limited Icon set?